decentraland/unity-explorer
# add track for launch from dcl-editor

## What does this PR change?

Extended `runtime` field of analytics context with the support of launch from `dcl-editor`

## How to test the changes?

1. Launch the explorer
2. Check happy path

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
